### PR TITLE
Upgrade converter to 1.0 format

### PR DESCRIPTION
**Pull request by Derstilon**
connected with https://github.com/yaptide/ui/pull/1042

- [X] update working json format to 1.0
- [X] add beam modulator parsing

### CODE REVIEW

File: converter/shieldhit/beam.py [hunk 1]
```diff
@@ -60,16 +114,26 @@ class BeamConfig:
...
+    beam_pos: tuple[float, float, float] = (0, 0, 0)  # [cm]
+    beam_dir: tuple[float, float, float] = (0, 0, 1)  # [cm]
     delta_e: float = 0.03  # [a.u.]
     nuclear_reactions: bool = True
+    
+    modulator_source_filename: Optional[str] = None
+    modulator_source_file_content: Optional[str] = None
+    modulator_zone_id: Optional[int] = None
+    modulator_simulation: ModulatorSimulationMethod = ModulatorSimulationMethod.MODULUS
+    modulator_mode: ModulatorInterpretationMode = ModulatorInterpretationMode.MATERIAL
```
Comment (grzanka) [inline converter/shieldhit/beam.py:127]: Why isn't that optional as well ?

**Comment by grzanka** [comment]
@Derstilon fix deepsource ...

File: converter/shieldhit/beam.py (hunk 1 above)
Comment (Derstilon) [reply converter/shieldhit/beam.py]: those have default values based on default values from documentation

**Comment by grzanka** [comment]
@Derstilon more deepsource :)

**Review by grzanka — APPROVED** [review]

**Pull request merged**